SQL/0000-00-00-schema.sql

```diff
@@ -368,6 +370,24 @@ CREATE TABLE `test_names` (
   CONSTRAINT `FK_test_names_1` FOREIGN KEY (`Sub_group`) REFERENCES `test_subgroups` (`ID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `test_json` (
```

Comment:
@driusan should we get rid of `test` for instruments and just call it... `instrument_JSON`

Reply:
i had thought about calling it instrument_JSON but seeing as it relates to test_names and belongs in the same group of tables, I wanted to keep uniform the naming